### ISSUE TITLE
Archive group as bookmark folder

### ISSUE THREAD

**Issue by igoralmeida** [Morikko/sync-tab-groups#161]
First of all, thanks for the great addon!

I am aware that bookmarks can be tricky and frustrating to work with, but I was wondering whether it would be possible to "archive" tab groups as bookmark folders. This way I can quickly remove several groups that I no longer use frequently but don't really want to delete.

**Comment by Morikko** [comment]
Yes, I like the idea. It is a nice feature I would even use myself.

I have not much time now, so feel free to propose a PR :)

**Comment by igoralmeida** [comment]
I'd need A LOT of handholding to get that to work. Maybe give some pointers for potential contributors in the future, it'll at least be documented somewhere.
